packages/simcore-sdk/src/simcore_sdk/node_ports_common/_filemanager.py

```diff
@@ -41,6 +41,13 @@ async def _get_location_id_from_location_name(
     raise exceptions.S3InvalidStore(store)
 
 
+def _get_secure_link(url: AnyUrl) -> str:
+    if is_storage_secure() and not f"{url}".startswith("https"):
+        return url.replace("http", "https")
```

Comment:
If you would happen to replace this `http://en.wikipedia.org/wiki/http`  it would be wrong. I know we are not in such a generic case but I would be very cautious because a bug like this is very tough to find in such a complex system

I would recommend:
-  using `AnyUrl` method (if any) that explicitly access and set the scheme?
- using `url.removeprefix(...)` or `url.replace(..., 1)` to savely remove only the prefix and not that word somewhere else

Reply:
good point, will replace